model/riscv_types.sail

```diff
@@ -318,13 +327,25 @@ function exceptionType_to_str(e) =
     E_Load_Page_Fault()    => "load-page-fault",
     E_Reserved_14()        => "reserved-1",
     E_SAMO_Page_Fault()    => "store/amo-page-fault",
+    E_Reserved_16()        => "reserved-2",
+    E_Reserved_17()        => "reserved-3",
+    E_SW_Check_Fault()     => "software-check-fault",
 
     /* extensions */
     E_Extension(e)         => ext_exc_type_to_str(e)
   }
 
 overload to_str = {exceptionType_to_str}
 
+/* SW check exception codes */
+enum SWCheckCodes = {LANDING_PAD_FAULT, SHADOW_STACK_FAULT}
```

Comment:
Shadow stack exceptions should be added when adding shadow stack support, not as part of landing pad support